manual_correction.py

```diff
@@ -756,7 +756,7 @@ def main():
                     time.sleep(0.1)
                     print("")
                     # build file names
-                    subject, ses, filename, contrast = utils.fetch_subject_and_session(file)
+                    subject, ses, filename, contrast, echo = utils.fetch_subject_and_session(file)
                     # Construct absolute path to the input file
                     # For example: '/Users/user/dataset/data_processed/sub-001/anat/sub-001_T2w.nii.gz'
                     fname = os.path.join(path_img, subject, ses, contrast, filename)
```

Comment:
Shouldn't the `echo` tag be included here?

Reply:
Okay, probably not, because it is already part of `filename`.

Reply:
It's strange because it seems to be there for me